tests/first.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import first from '../src/first';
+
+describe( 'utils', () => {
+	describe( 'first', () => {
+		it( 'should return first item', () => {
+			const iterator = [ 11, 22 ][ Symbol.iterator ]();
```

Comment:
Don't do too much in one line. The code above might be not easily readable for someone. I prefer:

```js
const collection = [ 11, 22 ];
const iterator = collection[ Symbol.iterator ]();
```